src/internal.c

```diff
@@ -7886,14 +7886,15 @@ static int DoUserAuthInfoRequest(WOLFSSH* ssh, byte* buf, word32 len,
         if (!prompts) {
             ret = WS_MEMORY_E;
         } else {
+            WMEMSET(prompts, '\0', sizeof(char*) * promptSz);
             echo = (byte*)WMALLOC(sizeof(byte) * promptSz, ssh->ctx->heap,
                                   DYNTYPE_BUFFER);
         }
 
         if (!echo) {
             ret = WS_MEMORY_E;
         } else {
-            WMEMSET(prompts, '\0', sizeof(char*) * promptSz);
+            WMEMSET(echo, 0, sizeof(byte) * promptSz);
```

Comment:
I can't think of a case where these memsets would be needed, but I suspect a static analyzer triggered it.